src/components/calcite-alert/calcite-alert.scss

```diff
@@ -1,3 +1,38 @@
+// theme variables
+// light theme
+:host {
+  --calcite-alert-background: #{$blk-000};
+  --calcite-alert-title-text: #{$blk-180};
+  --calcite-alert-message-text: #{$blk-160};
+  --calcite-alert-icon-fill: #{$blk-220};
+  --calcite-alert-close-background-hover: #{$blk-010};
+  --calcite-alert-close-background-pressed: #{$blk-020};
+  --calcite-alert-count-text: #{$blk-180};
+  --calcite-alert-count-border: #{$blk-010};
+  --calcite-alert-dismiss-background: #{rgba($blk-000, 0.6)};
+  --calcite-alert-border-blue: #{$ui-blue};
+  --calcite-alert-border-green: #{$ui-green};
+  --calcite-alert-border-red: #{$ui-red};
+  --calcite-alert-border-yellow: #{$ui-yellow};
+}
+
+// dark theme
+:host([theme="dark"]) {
+  --calcite-alert-background: #{$blk-200};
+  --calcite-alert-title-text: #{$blk-005};
+  --calcite-alert-message-text: #{$blk-010};
+  --calcite-alert-icon-fill: #{$blk-040};
+  --calcite-alert-close-background-hover: #{$blk-210};
+  --calcite-alert-close-background-pressed: #{$blk-220};
+  --calcite-alert-count-text: #{$blk-040};
+  --calcite-alert-count-border: #{$blk-210};
+  --calcite-alert-dismiss-background: #{rgba($blk-200, 0.6)};
+  --calcite-alert-border-blue: #{$ui-blue-dark};
```

Comment:
this is nice how the dark ui variables map so cleanly here ✨

Reply:
awesome !!!